datarepo-clienttests/src/main/java/scripts/testscripts/RetrieveSnapshot.java

```diff
@@ -151,22 +155,38 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
     snapshotSummaryModel =
         DataRepoUtils.expectJobSuccess(
             repositoryApi, createSnapshotJobResponse, SnapshotSummaryModel.class);
+
+    repositoryApi.addSnapshotPolicyMember(
+        snapshotSummaryModel.getId(),
+        "steward",
+        new PolicyMemberRequest().email("nmalfroy.dev@gmail.com"));
     logger.info(
         "Successfully created snapshot: {} with user {} ",
         snapshotSummaryModel.getName(),
         datasetCreator.name);
+    drsId = "v1_" + snapshotSummaryModel.getId() + "_" + fileId;
```

Comment:
Yeah, this what I like about this approach is that if we do, it's a further reminder to alert folks that the format is changing since this is a public interface

Reply:
I agree that we should test the format of DRS ids to ensure that they're stable and only change when we expect them to. I don't agree that any test that creates a DRS id should do this. This invariant only needs to be tested by the code that tests the class that generates DRS ids. Code that incidentally needs to generate an id shouldn't duplicate the code.